tests/views/test_admin_repo_index.py

```diff
@@ -0,0 +1,301 @@
+        repo_conf = config.Configuration('haiti')
+        self.assertIs(repo_conf.test_mode, True)
+        self.assertEqual(
+            repo_conf.updated_date,
+            utils.get_timestamp(datetime.datetime(2019, 5, 10, 12, 15, 0)))
+
+    def test_edit_keywords_config(self):
+        self.login_as_superadmin()
+        self._post_with_params(keywords='haiti,earthquake')
+        repo_conf = config.Configuration('haiti')
+        self.assertEqual(repo_conf.keywords, 'haiti,earthquake')
+
+    def test_edit_forms_config(self):
+        self.login_as_superadmin()
+        self._post_with_params(
+            use_family_name='true',
+            family_name_first='true',
+            use_alternate_names='true',
+            use_postal_code='true',
+            allow_believed_dead_via_ui='true',
+            min_query_word_length='2',
+            show_profile_entry='true')
+        repo_conf = config.Configuration('haiti')
+        self.assertIs(repo_conf.use_family_name, True)
+        self.assertIs(repo_conf.family_name_first, True)
+        self.assertIs(repo_conf.use_alternate_names, True)
+        self.assertIs(repo_conf.use_postal_code, True)
+        self.assertIs(repo_conf.allow_believed_dead_via_ui, True)
+        self.assertIs(repo_conf.min_query_word_length, 2)
+        self.assertIs(repo_conf.show_profile_entry, True)
+
+    def test_edit_map_config(self):
+        self.login_as_superadmin()
+        self._post_with_params(
+            map_default_zoom='8',
+            map_default_center='[32.7, 85.6]',
+            map_size_pixels='[300, 450]')
+        repo_conf = config.Configuration('haiti')
+        self.assertEqual(repo_conf.map_default_zoom, 8)
+        self.assertEqual(repo_conf.map_default_center, '[32.7, 85.6]')
+        self.assertEqual(repo_conf.map_size_pixels, '[300, 450]')
+
+    def test_edit_timezone_config(self):
+        self.login_as_superadmin()
+        self._post_with_params(
+            time_zone_offset='5.75',
+            time_zone_abbreviation='NPT')
+        repo_conf = config.Configuration('haiti')
+        self.assertEqual(repo_conf.time_zone_offset, 5.75)
+        self.assertEqual(repo_conf.time_zone_abbreviation, 'NPT')
+
+    def test_api_access_control_config(self):
+        self.login_as_superadmin()
+        self._post_with_params(
+            search_auth_key_required='true',
+            read_auth_key_required='true')
+        repo_conf = config.Configuration('haiti')
+        self.assertIs(repo_conf.search_auth_key_required, True)
+        self.assertIs(repo_conf.read_auth_key_required, True)
+
+    def test_edit_zero_rating_config(self):
+        self.login_as_superadmin()
+        self._post_with_params(zero_rating_mode='true')
+        repo_conf = config.Configuration('haiti')
+        self.assertIs(repo_conf.zero_rating_mode, True)
+
+    def test_edit_spam_config(self):
+        self.login_as_superadmin()
+        self._post_with_params(bad_words='voldemort')
```

Comment:
just checking, is testing with any string good enough, or would it be worth checking with multiple words?

Reply:
Here it's fine to test with any string, because the value is getting stored as-is; the code under test here isn't expected to parse it.